web/src/views/ReposOwner.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <FluidContainer class="flex flex-col">
+    <div class="flex flex-row flex-wrap border-b pb-4 mb-4 items-center dark:border-dark-200">
+      <h1 class="text-xl text-gray-500">{{ repoOwner }}</h1>
+      <TextField v-model="search" class="w-auto md:ml-auto" placeholder="Search ..." />
+      <Button class="md:ml-auto" :to="{ name: 'repo-add' }" start-icon="plus" text="Add repository" />
```

Comment:
I think we can remove this button from this page. As it could be confusing to a user having a look at some public org / other user and seeing a "I can add something to your org" button.

```suggestion
```